GNUmakefile

```diff
@@ -68,12 +68,11 @@ contracts-operator-ui-build: # only compiles tsc and builds contracts and operat
 	yarn setup:chainlink
 	CHAINLINK_VERSION="$(VERSION)@$(COMMIT_SHA)" yarn workspace @chainlink/operator-ui build
 
-.PHONY: abigen
-abigen:
+tools/bin/abigen:
```

Comment:
I don't see a reason to not use the makefile caching, especially since we have an explicit version check elsewhere